ModelCatalogueGenomicsPlugin/grails-app/controllers/org/modelcatalogue/core/genomics/GenomicsController.groovy

```diff
@@ -146,23 +145,24 @@ class GenomicsController {
 
 
     private void exportEligibilityOrPhenotypesAndTests(boolean eligibilityMode) {
-        DataClass model = DataClass.get(params.id)
+        DataClass dClass = DataClass.get(params.id)
+
+        DataClass latestVersion = (DataClass) elementService.findByModelCatalogueId(DataClass, dClass.getDefaultModelCatalogueId(true))
```

Comment:
I can see where you're heading but this may confuse users a lot. what if you want to export older version and you will always get the latest?